README.md

```diff
@@ -165,5 +122,4 @@ So, we add environment variable ENABLE_PRINT_EVENT_STDOUT to enable it.
 
 In default, it does not print the cluster event on the terminal when you execute the case on your terminal.
 
-if you want it for debugging, **please set `export ENABLE_PRINT_EVENT_STDOUT=true` before executing the case.**
-if you want it for debugging, **please set `export ENABLE_PRINT_EVENT_STDOUT=true` before executing the case.**
+if you want to enable it for helping debug, **please set `export ENABLE_PRINT_EVENT_STDOUT=true` before executing the case.**
```

Comment:
S/ if you want to enable it for helping debug/ If you would like to enable debugging for a case

Reply:
Addressed.